docs/source/operations/transformations/vertoffset.rst

```diff
@@ -0,0 +1,61 @@
+.. _vertoffset:
+
+================================================================================
+Vertical Offset And Slope
+================================================================================
+
+.. versionadded:: 9.1.0
+
+The Vertical Offset And Slope offsets transformation adds an offset to an
+orthometric height, taking account a constant offset and a inclinated plane,
+defined by its slope along latitude and longitude axis.
+
++---------------------+----------------------------------------------------------+
+| **Alias**           | vertoffset                                               |
++---------------------+----------------------------------------------------------+
+| **Domain**          | 3D                                                       |
++---------------------+----------------------------------------------------------+
+| **Input type**      | Geodetic coordinates (horizontal), meters (vertical)     |
++---------------------+----------------------------------------------------------+
+| **output type**     | Geodetic coordinates (horizontal), meters (vertical)     |
++---------------------+----------------------------------------------------------+
+
+It is documented as coordinate operation method code 1046 in the EPSG dataset (:cite:`IOGP2018`).
+It is typically used in Europe, to relate national vertical systems to
+pan-European vertical systems (EVRF200, EVRF2007).
```

Comment:
I would like to see a more in-depth description of the transformation here. It's too much of a faf to have to look up what 1046 means in the EPSG-world.

Reply:
there's isn't much text to borrow from the description of the EPSG dataset. I've added a Formula section which is the "objective" description of what this method does

Reply:
Thanks, I think that puts the docs in a good place